packages/leaflet-map/src/components/leaflet/leaflet-map.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component } from "@ribajs/core";
+
+import template from "./leaflet-map.component.html";
+import * as L from "leaflet";
+import markerShadow from "leaflet/dist/images/marker-shadow.png";
```

Comment:
Hat sich TypeScript bei dem Typ `.png` gar nicht beschwert? Ansonsten müssten wir den Typ noch hinterlegen. Ich hatte versucht für sowas ein packages `@ribajs/types` zu hinterlegen aber war daran gescheitert weswegen sie vorerst in `packages/core/src/types/templates.ts` hinterlegt sind

Reply:
Bei mir nicht, das ist aber genrell etwas komisch bei mir.

Reply:
Dann sollten wir uns das noch einmal anschauen, ansonsten hast du gar nicht die Vorteile die das beim entwicklen bietet